corbel/src/main/scala/graffiti/corbel/auth/AccessRuleAuthorization.scala

```diff
@@ -0,0 +1,29 @@
+package graffiti.corbel.auth
+
+import com.google.gson.JsonObject
+import spray.http.HttpRequest
+import spray.routing.RequestContext
+
+/**
+ * @author Alexander De Leon <me@alexdeleon.name>
+ */
+class AccessRuleAuthorization(authorizationInfo: AuthorizationInfo) extends (RequestContext => Boolean)
+  with CorbelAuth {
+
+  def apply(requestContext: RequestContext): Boolean = {
+    implicit val request = requestContext.request
+    authorizationInfo.accessRules.forall(validate)
```

Comment:
authorizationInfo.accessRules.exists(validate)

The forall method tests whether a predicate holds for all elements of this iterable collection.